_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -974,13 +975,21 @@ public static function update_module( $data ) {
 				break;
 
 			case 'post_by_email_address':
-				$post_by_email = new Jetpack_Post_By_Email();
 				if ( 'create' == $value ) {
-					$result = $post_by_email->create_post_by_email_address();
+					$result = self::_process_post_by_email(
```

Comment:
should these three all carry the same error?  Maybe a different error code at the end to make troubleshooting easier?